sdk/servicebus/service-bus/src/util/parseUrl.ts

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+const url = require("url");
+
+export const parseURL = url.parse;
```

Comment:
`url.parse` is part of the legacy API in node.js. The `url.URL` class implements the same WHATWG URL standard that the browser `URL` does, so we should use that instead since then we'll get the same parsing behavior between platforms.

Reply:
Updated - https://www.npmjs.com/package/url didn't seem to point anything about this being deprecated. Is there a resource that talks about the differences as such?

Reply:
We aren't using the npm package `url`, it's one of the modules built into node.js:
https://nodejs.org/dist/latest-v12.x/docs/api/url.html#url_legacy_url_api

Reply:
I see, good to know - thanks!

Created a new branch off master and reverted the feature disabling commit - hopefully this should add in the CI fix from master. cc: @ramya-rao-a

Reply:
I see that the entire commit of #6320 has been reverted with https://github.com/Azure/azure-sdk-for-js/commit/1f9551ef950fed2f755980646fbfaf18e2a57925 and the resulting branch `feature/service-bus-atom-p1` is being used as the base branch instead of `feature/service-bus-atom`. 

But the revert commit is also touching other files unrelated to the atom api feature like the version update in package.json file and the constants file.

Please consider the below follow ups to keep the changes and branch list clean
- Add back the changes to the package.json and the constants file i.e change the version back to 1.1.1 in the `feature/service-bus-atom-p1` branch
- Delete the older branch `feature/service-bus-atom`. Now that you plan to use the new branch as the base, there is no need for the older branch to exist
- Remove the suffix `-p1` from the base branch. It is not clear as to what the suffix is suppose to represent where as the name `feature/service-bus-atom` is very self explanatory.